### ISSUE TITLE
Unable to add a topic for a label with a specific pattern

### ISSUE THREAD

**Issue by mzirlinger** [neo4j-contrib/neo4j-streams#382]
## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use Streams, please ask on [StackOverflow](https://stackoverflow.com/questions/tagged/neo4j-streams) instead of creating an issue here.

## Expected Behavior (Mandatory)
Topic creation for specific label

## Actual Behavior (Mandatory)
Topic configuration fails to create because of unusual label naming

## How to Reproduce the Problem
The neo4j instance has some unusual label names that can be escaped with backticks in cypher. However, I have been unable to successfully escape and get the label names to pass the routing configuration regex in neo4jstreams.
Example: NEO4J_streams_source_topic_nodes_my_topic: "Label:\`labels::label\`{*}". 

The :: within the label name breaks the topic creation. I have tried various combinations along the lines of

\`labels::label\`{*}
labels::label{\*}

### Simple Dataset (where it's possible)

```
//Insert here a set of Cypher statements that helps us to reproduce the problem

```

### Steps (Mandatory)

  1. Spin up docker-compose for neo4j locally
  1. Set topic env variables in compose file. 
  1. neo4j will error out on topic configuration for label

## Specifications (Mandatory)

Currently used versions

### Versions

  - OS: macOS catalina
  - Neo4j: 3.5.2-enterprise
  - Neo4j-Streams: 3.5.12

**Comment by mzirlinger** [comment]
Hey There. I was just wondering when we think this will be publicly available/merged?

**Comment by conker84** [comment]
closed via 21142e7
